### ISSUE TITLE
Automatic activation

### ISSUE THREAD

**Issue by fbruetting** [bwinton/TabCenter#1055]
I want to suggest an option, that shows new Firefox windows with a horizontal tab bar first, and then switches to the vertical side bar, when either the horizontal bar is full, or when the window has a certain width. The first criterion ensures better use of the screen space (so you don't have a large side panel while having just two tabs open), while the second criterion enables the side bar, when the additional window width is of no more use. What do you think about this?

**Comment by ericawright** [comment]
just to confirm that I'm understanding correctly. The user would be able to choose "auto", and then in the middle of resizing their window the tabs would jump to the side or top?
I suppose a few people might like it.

**Comment by fbruetting** [comment]
Yes. If the window is too thin, the side bar blocks useful screen space and should be automatically changed to the top bar. In contrast to this, if the window is really wide and web pages can make no use of all the screen space, the side panel should automatically be displayed instead of the top bar.

A further factor can be the number of open tabs, because you don't need a side bar for three open tabs, but 50 tabs in the top bar is crazy.
